src/Kregielnia.hpp

```diff
@@ -1,5 +1,5 @@
 #pragma once
-#include "Helper.hpp"
+#include "TestFileSupport.hpp"
```

Comment:
If TestFileSupport is created for testing purposes only it should not be included in any source or header file. Implementation should not be aware of testing procedures.

Reply:
@LordLukin  In my understanding there is no task related to create input files for project. So this files should exist only in test domain. I can imagine that this project is only part of bigger system. So we can test this 'Kregielnia project' regarding to requirements but we can not run this project because we do not have input files. So my understanding is that we should prepare proper test but we can not run this program because we do not have input files. Is it correct?

Reply:
Yes, main.cpp is not important right now. You can put some use case there if you wish.